docker/child.conf

```diff
@@ -36,7 +36,7 @@ LinkPass "LiNk"
 # Normally you want to load all these modules
 LoadMod "m_help m_bot m_chan m_host m_nick m_oper"
 # You can load additionnal modules, such as secserv, spamscan, or connect4
-LoadMod "secserv"
+LoadMod "sasl secserv"
```

Comment:
Where is this sasl module coming from?

Reply:
Left-over from another branch, removed it (along with some libprom references in unreal's dockerfile)